wp-sw-cache/wp-sw-cache-db.php

```diff
@@ -5,20 +5,20 @@ class SW_Cache_DB {
   private static $instance;
   public static $options = array(
     // For v1 we'll prompt the user enable the plugin manually upon activation
-    'wp_sw_cache_enabled' => 0,
+    'wp_sw_cache_enabled' => 1,
     // The "style.css" file is a standard WordPress file, so we can safely assume this exists
     'wp_sw_cache_files' => array('styles.css'),
     // Create an initial SW version
-    'wp_sw_cache_version' => '',
+    'wp_sw_cache_version' => '0.1.0',
```

Comment:
I also want to manually require files because WordPress sometimes moves functions aroudn